langchain/callbacks/manager.py

```diff
@@ -425,10 +436,13 @@ async def on_llm_error(
 class CallbackManagerForChainRun(RunManager, ChainManagerMixin):
     """Callback manager for chain run."""
 
-    def get_child(self) -> CallbackManager:
+    def get_child(self, tag: str | None = None) -> CallbackManager:
```

Comment:
It doesn't impact anything python and not w/in a dataclass but this syntax isn't really supported in py 3.8

Reply:
Cool, fixed